usr/sbin/pkcsconf/pkcsconf.c

```diff
@@ -1072,6 +1120,14 @@ CK_RV init(void)
     if (rc != CKR_OK) {
         printf("Error initializing the PKCS11 library: 0x%lX (%s)\n", rc,
                p11_get_ckr(rc));
+
+        if (check_user_and_group() != CKR_OK) {
+            printf("Note: all non-root users that require access to PKCS#11 "
```

Comment:
consider using fprintf(stderr, ...
however, it looks like the other errors are also printed to stdout instead of stderr.
So if this is a bigger change, let's go with stdout then.